cpp/include/raft/detail/span.hpp

```diff
@@ -86,5 +86,30 @@ __host__ __device__ constexpr auto lexicographical_compare(InputIt1 first1,
   }
   return first1 == last1 && first2 != last2;
 }
+
+template <typename T, typename SizeType, size_t Extent>
```

Comment:
I assume the `Extent` should be of `SizeType`? 
```suggestion
template <typename T, typename SizeType, SizeType Extent>
```

Reply:
Removed the `SizeType`.